bin/send-comments

```diff
@@ -155,6 +155,10 @@ while ( my $body = $bodies->next ) {
                 send_fail_timestamp => \'current_timestamp',
                 send_fail_reason => 'Failed to post over Open311',
```

Comment:
Might be good to copy SendReport/Open311.pm here and put the Open311 error in `send_fail_reason` (it puts this string plus the open311->error). Then we can always see the error in the DB too as well as in email.